.github/workflows/static.yml

```diff
@@ -11,6 +11,15 @@ jobs:
           go-version: 1.15
       - run: go mod download
       - run: go vet ./...
+  mod:
+    name: mod
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.15
+      - run: go mod tidy && test -z "$(git status --porcelain go.sum go.mod)"
```

Comment:
Could you split this into a separate PR?

Reply:
https://github.com/smartcontractkit/chainlink/pull/3568/files